.github/workflows/ci.yml

```diff
@@ -137,6 +137,24 @@ jobs:
       - id: set-test-module-names
         name: Set modules tests names
         run: echo "::set-output name=test-module-names::$(npx jest **/modules/* --listTests --json | jq -c 'map(split("/") | .[-1] | split(".") | .[0] | gsub( "-"; "_") | ascii_upcase | tostring)')"
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Build and push
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          push: false
+          load: true
+          cache-from: type=gha
+          cache-to: type=gha
```

Comment:
what does `type=gha` mean?

Reply:
It means [Github Actions Cache](https://docs.docker.com/build/building/cache/backends/gha/) which its contents are available at [this page](https://github.com/organizations/iasql/settings/actions/caches).

Different backends are discussed [here](https://docs.docker.com/build/building/cache/backends/#backends).